pyanaconda/ui/__init__.py

```diff
@@ -163,13 +177,22 @@ def _orderActionClasses(self, spokes, hubs):
                      attribute of Spokes to pick up
         :type hubs: common.Hub based types
         """
+        ordered_spokes = sorted(spokes, key=lambda x: x.__name__)
+        action_classes = []
 
-        actionClasses = []
         for hub in hubs:
-            actionClasses.extend(sorted(filter(lambda obj, h=hub: getattr(obj, "preForHub", None) == h, spokes),
-                                        key=lambda obj: obj.priority))
-            actionClasses.append(hub)
-            actionClasses.extend(sorted(filter(lambda obj, h=hub: getattr(obj, "postForHub", None) == h, spokes),
-                                        key=lambda obj: obj.priority))
-
-        return actionClasses
+            action_classes.extend(
+                sorted(
+                    filter(lambda obj, h=hub: getattr(obj, "preForHub", None) == h,
```

Comment:
I don't think this is readable. I would replace the lambda function with a function. It looks like it could be used also in tests. Same below.

Reply:
Fixed.